example-digiwf-email-integration/src/main/java/io/muenchendigital/digiwf/email/integration/api/controller/ExampleController.java

```diff
@@ -47,9 +47,13 @@ private Mail getMail() {
         mail.setSubject("Test1234");
         mail.setBody("Hallo test123");
         mail.setReplyTo("");
-//        final List<String> attachmentPaths = new ArrayList<>();
-//        attachmentPaths.add("test/picture.jpg");
-//        mail.setAttachmentPaths(attachmentPaths);
+//        Attachment
```

Comment:
remove commented out code

Reply:
Moved to own method, only left one commented out code in with a comment to comment it in if needed.